projectgenerator/gui/generate_project.py

```diff
@@ -64,6 +64,13 @@ def __init__(self, parent=None):
     def accepted(self):
         configuration = self.updated_configuration()
 
+        if not configuration.host:
+            self.txtStdout.setText(self.tr('Please set a host before creating the project.'))
+            return
+        if not configuration.database:
+            self.txtStdout.setText(self.tr('Please set a database before creating the project.'))
+            return
```

Comment:
Done in https://github.com/opengisch/projectgenerator/pull/42/commits/ac4ca7932f8fb11267b3f97cda83a86fe4616c7f